services/backend.js

```diff
@@ -41,10 +41,14 @@ const STRINGS = {
     env_secret: `* Using env var for secret`,
     env_client_id: `* Using env var for client-id`,
     env_owner_id: `* Using env var for owner-id`,
+    env_owner_name: `* Using env var for owner-name`,
+    env_owner_name_lookup: `* Using env var for owner-name to lookup owner-id`,
     server_started: `Server running at %s`,
     missing_secret: "Extension secret required.\nUse argument '-s <secret>' or env var 'EXT_SECRET'",
     missing_clientId: "Extension client ID required.\nUse argument '-c <client ID>' or env var 'EXT_CLIENT_ID'",
     missing_ownerId: "Extension owner ID required.\nUse argument '-o <owner ID>' or env var 'EXT_OWNER_ID'",
+    ownerId_retreival_error: "Failed to find a Twitch profile for the ownerName %s",
```

Comment:
Thank you for adding the appropriate strings! I think there's typo here in the word 'retrieval' here. Would you mind fixing? Thanks!